src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/SelectionFieldFilterType.js

```diff
@@ -30,7 +31,11 @@ class SelectionFieldFilterType extends AbstractFieldFilterType<?Array<string | n
     ) {
         super(onChange, parameters, value);
 
-        this.selectionStore = new MultiSelectionStore(this.resourceKey, []);
+        this.selectionStore = new MultiSelectionStore(
+            this.resourceKey,
+            [],
+            computed(() => userStore.contentLocale)
```

Comment:
this is necessary because the `CategoryController` will throw an error if the request does not include any locale. unfortunately this change depends on #5805, because the `SearchStore` does not have any locale before that PR:
https://github.com/sulu/sulu/pull/5805/files#diff-9f4b5bb9b4570d8c7e2c3314b256e334a86bb15875300447290886166230715f

Because of this, this PR needs to target `2.x`.